### PR TITLE
usbutils: new package at v012

### PR DESCRIPTION
**Pull request by trbednarzyk**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package

USB utilities for Linux, including lsusb.

## Installed manifest of package

(`cat /var/db/kiss/installed/usbutils/manifest`)

```
/var/db/kiss/installed/usbutils/version
/var/db/kiss/installed/usbutils/sources
/var/db/kiss/installed/usbutils/manifest
/var/db/kiss/installed/usbutils/depends
/var/db/kiss/installed/usbutils/checksums
/var/db/kiss/installed/usbutils/build
/var/db/kiss/installed/usbutils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/usbhid-dump.8
/usr/share/man/man8/lsusb.8
/usr/share/man/man8/
/usr/share/man/man1/usb-devices.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/usbhid-dump
/usr/bin/usb-devices
/usr/bin/lsusb.py
/usr/bin/lsusb
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/usbutils/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/gregkh/usbutils/archive/v012.tar.gz
```
Comment (dylanaraps) [inline community/usbutils/sources]: https://www.kernel.org/pub/linux/utils/usb/usbutils/usbutils-012.tar.xz

This source should allow you to drop `autogen.sh`.

File: community/usbutils/version [hunk 2]
```diff
@@ -0,0 +1 @@
+v012 1
```
Comment (dylanaraps) [inline community/usbutils/version]: Change this to `012`.

File: community/usbutils/sources (hunk 1 above)
Comment (trbednarzyk) [reply community/usbutils/sources]: I changed the source to this (sans the www.) but there was not a `configure` file in the source. Unless I'm missing something `autogen.sh` will still need to be run with this source.

Comment (dylanaraps) [reply community/usbutils/sources]: Understood. :+1:

**Pull request merged**